night_of_the_dead/egg-pterodactyl-night-of-the-dead.json

```
@@ -57,7 +57,7 @@
             "default_value": "1",
             "user_viewable": false,
             "user_editable": false,
-            "rules": "required|boolean|in:1",
+            "rules": "boolean|in:1",
```

```suggestion
            "rules": "boolean",
```
I don't get the use of the `in:1` here `$( [[ \"${WINDOWS_INSTALL}\" == \"1\" ]]` could be replaced in the install script